config/sshdcond/sshdcond.inc

```diff
@@ -45,57 +41,59 @@ function sshdcond_custom_php_install_command() {
 
 	conf_mount_rw();
 
-	// We need to generate an outfile for our extra commands
-	// The patched g_szSSHDFileGenerate php file then reads and appends that config
+	/* We need to generate an outfile for our extra commands.
+	The patched g_szSSHDFileGenerate php file then reads and appends that config.
+	*/
 	$fd = fopen("/etc/ssh/sshd_extra", 'w');
 	fclose($fd);
 
 	conf_mount_ro();
-	}
+}
 
 function sshdcond_custom_php_deinstall_command() {
 	global $g, $config;
 
 	conf_mount_rw();
 
-	// 1. Delete our config file
+	/* Delete our config file. */
 	unlink_if_exists("/etc/ssh/sshd_extra");
 
-	// 2. Re-run sshd config generation script
+	/* Re-run sshd config generation script. */
 	restart_sshd();
 
 	conf_mount_ro();
-	}
+}
 
 function sshdcond_custom_php_write_config() {
-	global $g, $config;
+	global $g, $config, $pkg_interface;
 
-	# detect boot process
-	if (is_array($_POST)) {
-		if (!preg_match("/\w+/",$_POST['__csrf_magic']))
+	/* Detect boot process, do nothing during boot. */
+	if (platform_booting()) {
 			return;
```

Comment:
Yeah, it got broken b/c I tried ot do a simple change via web browser, appanrently horrible idea.